src/mds/ScrubStack.h

```diff
@@ -203,15 +203,15 @@ class ScrubStack {
    * @param added_children set to true if we pushed some of our children
    * @param done set to true if we started to do final scrub
```

Comment:
nit: update the comment with new argument.

Reply:
this new arg doesn't exist anymore, closing the conversation